src/tool_homedir.c

```diff
@@ -86,10 +71,18 @@ char *homedir(void)
  }
 #endif /* PWD-stuff */
 #ifdef WIN32
-  home = GetEnv("APPDATA", TRUE);
-  if(!home)
-    home = GetEnv("%USERPROFILE%\\Application Data", TRUE); /* Normally only
-                                                               on Win-2K/XP */
+  home = GetEnv("APPDATA");
+  if(!home) {
+    char *env = GetEnv("USERPROFILE");
```

Comment:
Why not put this into a const array and loop through it and break if a valid envvar was found instead of using a nested if/else statement?

Reply:
USERPROFILE must be appended to so it's %USERPROFILE%\\Application Data, it is not the same as APPDATA. However even if it was I don't think I'd loop it for just two.